hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveStatelessProxyUpdateTest.java

```diff
@@ -112,6 +109,11 @@ public void testLazyInitializationException(TestContext context) {
 							return s.update( entityFromDb );
 						} )
 				)
+				.onFailure().recoverWithUni( throwable -> {
```

Comment:
You can use assert thrown or this test won't fail in case we don't throw an exception

Reply:
refactored and fixed